pyomo/common/tests/test_timing.py

```diff
@@ -34,13 +34,13 @@ def test_report_timing(self):
         m = ConcreteModel()
         m.x = Var([1,2])
 
-        ref = """
-           0 seconds to construct Block ConcreteModel; 1 index total
-           0 seconds to construct RangeSet FiniteSimpleRangeSet; 1 index total
-           0 seconds to construct Var x; 2 indicies total
-           0 seconds to construct Suffix Suffix; 1 index total
-           0 seconds to apply Transformation RelaxIntegerVars (in-place)
-""".strip()
+        ref = r"""
+           (0(\.\d+)?) seconds to construct Block ConcreteModel; 1 index total
+           (0(\.\d+)?) seconds to construct RangeSet FiniteSimpleRangeSet; 1 index total
+           (0(\.\d+)?) seconds to construct Var x; 2 indices total
+           (0(\.\d+)?) seconds to construct Suffix Suffix; 1 index total
+           (0(\.\d+)?) seconds to apply Transformation RelaxIntegerVars \(in-place\)
+           """.strip()
```

Comment:
This needs to be:
```python
        ref = r"""
(0(\.\d+)?) seconds to construct Block ConcreteModel; 1 index total
 +(0(\.\d+)?) seconds to construct RangeSet FiniteSimpleRangeSet; 1 index total
 +(0(\.\d+)?) seconds to construct Var x; 2 indices total
 +(0(\.\d+)?) seconds to construct Suffix Suffix; 1 index total
 +(0(\.\d+)?) seconds to apply Transformation RelaxIntegerVars \(in-place\)
           """.strip()
```
becayuse the inter-line whitespace will change if the time moves from `0` to `0.01`